packages/ckit/src/tx-builders/ChequeWithdrawBuilder.ts

```diff
@@ -37,9 +37,20 @@ export class ChequeWithdrawBuilder extends AbstractPwSenderBuilder {
     };
     const unwithdrawnCells = await provider.collectCells({ searchKey }, (cells) => cells.length <= 1000);
 
+    const currEpoch = await provider.rpc.get_current_epoch();
+    const validEpoch = Number(currEpoch.number) - 6;
+    const validWithdrawCells: Cell[] = [];
+    for (const cell of unwithdrawnCells) {
+      const cellEpoch = Number(
+        '0x' + (await provider.rpc.get_block_by_number(cell.block_number!, '0x2'))?.header.epoch!.substring(9),
```

Comment:
we should request RPC via [batch](https://www.jsonrpc.org/specification#batch) mode to speed up request

Reply:
After discussing with Ximing, we think it can’t request rpc via batch mode under current implementation.
There are two possible solutions:
1. Extend RPC `get_xx` function so that it can support batch mode.
2. Add `getRpcUrl` function in `MercuryProvider`, and then write another batch request function just for `ChequeWithdrawBuilder`

I prefer the second one. Because the first one needs to change other `get_xx` functions in `lumos/rpc` as well, and maybe `ckb-js-toolkit`, I guess. It’s more universal but the second one is much easier…

Reply:
maybe we can downgrade `MercuryProvider`'s constructor to construct with RPC URL only like this `constructor(private mercuryRpc: string, private ckbRpc: string)` and add a function `batchRequestCkb(request: RpcRequest[])`

```ts
interface JsonRpcRequest {
  method: string;
  params: unknown;
}

class MercuryClient {
  constructor(private mercuryRpc: string, private ckbRpc: string) {
      ...
  }

  batchRequestCkb<T>(request: JsonRpcRequest): T {
  }
}
```

Reply:
this would be another feature, we can open another PR, so I merged this PR first